src/blog/newsletter-jan22.md

```diff
@@ -0,0 +1,21 @@
+Welcome to the first Flowforge newsletter, we’re going to publish this as a regular roundup of what\`s happening with both Flowforge and the wider Node-RED community, if you want to receive it via email, sign up for updates at the bottom of the page.
```

Comment:
- Can we rename the file to `community-news-01.md` ? Gives us more latitude than putting a date in the name.
 - We need metadata at the top of the post for this to get posted.
```
---
title: Community News #1
subtitle: News from the FlowForge and Node-RED communities
description: News from the FlowForge and Node-RED communities
date: 2022-01-28
authors: ["sam-machin"]
---
```
 - `FlowForge` not `Flowforge`